server/src/main/java/org/elasticsearch/common/logging/RateLimitingFilter.java

```diff
@@ -35,13 +36,14 @@
  * This filter works by using a lruKeyCache - a set of keys which prevents a second message with the same key to be logged.
  * The lruKeyCache has a size limited to 128, which when breached will remove the oldest entries.
  *
- * It is possible to disable use of `x-opaque-id` as a key with {@link RateLimitingFilter#setUseXOpaqueId(boolean) }
+ * It is possible to disable use of `x-opaque-id` as a key with {@link RateLimitingFilter#setUseXOpaqueIdEnabled(boolean) }
  * @see <a href="https://logging.apache.org/log4j/2.x/manual/filters.htmlf">Log4j2 Filters</a>
  */
 @Plugin(name = "RateLimitingFilter", category = Node.CATEGORY, elementType = Filter.ELEMENT_TYPE)
 public class RateLimitingFilter extends AbstractFilter {
-
-    private volatile boolean useXOpaqueId = true;
+    private static final String KIBANA_OPRDUCT_ORIGIN = "kibana";
```

Comment:
is this a typo or intentional?
```suggestion
    private static final String KIBANA_PRODUCT_ORIGIN = "kibana";
```

Reply:
that was a typo. The constant is not needed as this change will apply to all elastic originating requests